modules/ossm-rn-deprecated-features.adoc

```diff
@@ -13,6 +13,10 @@ Some features available in previous releases have been deprecated or removed.
 
 Deprecated functionality is still included in {product-title} and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments.
 
+== Deprecated features {ProductName} 2.0.8
+
+To remain compatible with {product-title} 4.8 the `apiextensions.k8s.io/v1beta1` API was deprecated as of {ProductName} 2.0.8.
```

Comment:
Just want to confirm with dev team that the correct version of OpenShift to list here is 4.8, the most recent release.

Reply:
The API change is for the 4.9 compatibility. However,  4.9 has not been released. So it's fine you put 4.8 there.

Reply:
Thank you for confirming!

Reply:
I would move this to fixed issues.  This was deprecated in OpenShift, not Service Mesh, and is being removed in OpenShift 4.9.  We don't provide any direct support for these APIs, but we do make use of them, which is why we needed to update the release.  The text might then read:
```suggestion
Updated use of `apiextensions.k8s.io` API from `v1beta1` to `v1` to ensure future versions of {product-title} are supported.
```

Reply:
@rcernich when I chatted with Julie about this topic, she had a concern that some customers might have created custom code that calls the API, and that the code would break. In that line of thought, she felt strongly that this belonged in the deprecated section. If it's correct that custom code could break, is listing this as fixed is the best location? I just want to ask to help ensure we're thinking of the customer experience perspective.